dbms/src/Disks/DiskSpaceMonitor.h

```diff
@@ -126,6 +126,9 @@ class StoragePolicy
     /// Reserve space on any volume with index > min_volume_index
     ReservationPtr reserve(UInt64 bytes, size_t min_volume_index) const;
 
+    /// Reserve space on any volume iterating them in reverse order
+    ReservationPtr reserveSlowest(UInt64 bytes) const;
```

Comment:
really bad name for several reasons.

Reply:
@filimonov Can you advise how good name may look like please?

Reply:
No good ideas. reserveOnLowestPriorityVolume :D
And it looks like it can be private (the problem of bad name will not be so important there)

Reply:
Basically it is not needed at all because you are right in the other thread.